### PR TITLE
docs: Register Mfa guide

### PR DESCRIPTION
**Pull request by hifabienne**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Comment by muhlemmer** [comment]
@hifabienne  Can you please fix the Vercel deployment, so we can review the rendered docs?

**Comment by hifabienne** [comment]
> @hifabienne Can you please fix the Vercel deployment, so we can review the rendered docs?

done

File: docs/docs/guides/integrate/login-ui/mfa.mdx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+import MfaOptions from './_list-mfa-options.mdx';
+import BrowserRegisterWebAuthN from './_browser_register_webauthn.mdx';
+
+
+Multi-factor authentication (MFA) is a multi-step account authentication which requires to user to enter more than only the password.
+It is highly recommended to use MFA or [Passkeys](./passkey) to make your user accounts more secure.
+
+ZITADEL supports two different Methods:
+- Time-based one time password (TOTP), which are Authenticator apps like Google/Microsoft Authenticator, Authy, etc
+- Universal Second Factor (U2F), which is authentication with your device like Windows Hello, Apple FaceID, Fingerprint, etc.
```
Comment (muhlemmer) [inline docs/docs/guides/integrate/login-ui/mfa.mdx:15]: Do we need to mention FIDO2 keys, just for the sake of search results?

**Review by muhlemmer — COMMENTED** [review]
Looks really good. Left just a small question.

File: docs/docs/guides/integrate/login-ui/mfa.mdx (hunk 1 above)
Comment (hifabienne) [reply docs/docs/guides/integrate/login-ui/mfa.mdx]: Makes sense, I added Fido2 keys and yubikey as they are probably the most common

**Review by muhlemmer — APPROVED** [review]

**Pull request merged**